.github/workflows/create-charts.sh

```diff
@@ -75,14 +75,54 @@ update_platform_services_charts() {
         echo '    condition: kafka-topic-creator.enabled'
 } 
 
+write_to_tmp_source() {
+        echo 'attribute-service:'$attribute_service_version
+        echo 'gateway-service:'$gateway_service_version
+        echo 'query-service:'$query_service_version
+        echo 'entity-service:'$entity_service_version
+        echo 'config-service:'$config_service_version
+        echo 'hypertrace-graphql:'$hypertrace_graphql_version
+        echo 'hypertrace-ui:'$hypertrace_ui_version
+        echo 'hypertrace-ingester:'$hypertrace_ingester_version
+}
+
+write_changelog() {
+    echo '### Attribute service ('$(cat "kubernetes/tmp/compatibility_matrix.txt" | grep "^attribute-service:" | cut -d":" -f2-) '--->' $attribute_service_version')'
```

Comment:
Can we do this as below?

```
attribute_service_piror_version=get_prior_service_version('attribute-service')
    echo '### Attribute service ('$attribute_service_piror_version '--->' $attribute_service_version')'
    changelog -m hypertrace attribute-service $attribute_service_piror_version $attribute_service_version
```

and having a common function to get prior version

```
get_prior_service_version(service_name) {
  return $(cat "kubernetes/tmp/compatibility_matrix.txt" | grep "^${service_name}:" | cut -d":" -f2-)
}
```